File: C++/Basic/arithmetic.cpp
#include <iostream>

int main() {
    int a = 10, b = 5;
    std::cout << "Addition: " << (a + b) << std::endl;
    std::cout << "Subtraction: " << (a - b) << std::endl;
    std::cout << "Multiplication: " << (a * b) << std::endl;
    std::cout << "Division: " << (a / static_cast<double>(b)) << std::endl; // Use static_cast for floating-point division
    return 0;
}

File: C#/Basic/arithmetic.cs
using System;

class Program
{
    static void Main()
    {
        int a = 10, b = 5;
        Console.WriteLine("Addition: " + (a + b));
        Console.WriteLine("Subtraction: " + (a - b));
        Console.WriteLine("Multiplication: " + (a * b));
        Console.WriteLine("Division: " + (a / (double)b)); // Cast to double for accurate division
    }
}
